DigitalLearningSolutions.Data.Tests/DataServices/UserDataServiceTests/DelegateUserDataServiceTests.cs

```diff
@@ -54,6 +54,28 @@ public void GetDelegateById_returns_delegate_with_correct_user_centre_details()
             returnedDelegateEntity.UserCentreDetails.EmailVerified.Should().Be(expectedUserCentreDetails.EmailVerified);
         }
 
+        [Test]
+        public void GetDelegateById_returns_delegate_with_correct_adminId()

```

Comment:
Worth writing a test to check it doesn't return an adminId if the admin account is inactive.